g2g.go

```diff
@@ -29,14 +34,28 @@ type namedVar struct {
 	v    expvar.Var
 }
 
+// SplitEndpoint splits the provided endpoint string into its network and address
+// parts. It will default to 'tcp' network to ensure backward compatibility when
+// the endpoint is not prefixed with a network:// part.
+func SplitEndpoint(endpoint string) (string, string) {
```

Comment:
This probably doesn't need to be an exported function, since it's internal to the package and shouldn't be needed by a user.